### PR TITLE
feat: adds striped bg to palette view

### LINKED ISSUE #37: Add candy stripe bg to added colors area

### PR DESCRIPTION
**Pull request by schne324**
Closes #37

### CODE REVIEW

**Comment by schne324** [comment]
candy stripe bg
candy strip bg (mobile view)

**Comment by schne324** [comment]
candy strip bg
candy strip bg (mobile)

File: src/components/Palette/index.css [hunk 1]
```diff
@@ -1,3 +1,17 @@
...
+    152deg,
+    #f9f9f9 25%,
+    #ffffff 25%,
+    #ffffff 50%,
+    #f9f9f9 50%,
+    #f9f9f9 75%,
+    #ffffff 75%,
+    #ffffff 100%
+  );
+  background-size: 102.24px 54.36px;
```
Comment (jkodu) [inline src/components/Palette/index.css]: The size seems a bit too strict, otherwise 👍

Comment (schne324) [reply src/components/Palette/index.css]: The only way I know how to do these striped gradients are with some very specific `background-size` values. Useful to note that this doesn't restrict the element to that size.

Feel free to suggest an alternative if you've got one

**Pull request merged**